ledger-tool/src/main.rs

```diff
@@ -2727,7 +2730,22 @@ fn main() {
                     output_directory.display()
                 );
 
+                let mut accounts_index_config = AccountsIndexConfig::default();
```

Comment:
The below block is all duplicate code with what is in `verify` subcommand.

There is a nicer / less duplicate lines fix in master already with https://github.com/solana-labs/solana/pull/29975, so just doing the most basic thing in the v1.14 branch to minimize the diff